targets/targets.json

```diff
@@ -534,10 +534,21 @@
         "is_disk_virtual": true,
         "inherits": ["Target"],
         "detect_code": ["0218"],
-        "device_has": ["ANALOGIN", "ANALOGOUT", "ERROR_RED", "I2C", "I2CSLAVE", "INTERRUPTIN", "PORTIN", "PORTINOUT", "PORTOUT", "PWMOUT", "RTC", "SEMIHOST", "SERIAL", "SLEEP", "SPI", "SPISLAVE", "STDIO_MESSAGES"],
```

Comment:
Agreed. @0xc0170 I'll make this use target inheritance instead. @thinkberg I'm going to add another commit to the series. If you think I should remove it, I can do so.